server/grpc/interceptor/mocks_test.go

```diff
@@ -0,0 +1,224 @@
+package interceptor
```

Comment:
We use a [minimock + a generator](https://github.com/instill-ai/pipeline-backend/tree/main/pkg/mock). And group mocks in a single package. I don't have a strong opinion about putting the mocks next to their interface in `x`, but I'd align with the generation tool and flow